### DISCUSSION TITLE
cargo +1.72.1 build --release --target=x86_64-unknown-linux-musl error: could not compile `qdrant` (bin "qdrant") due to previous error

### DISCUSSION THREAD

**Discussion by MaksimBy** [qdrant/qdrant#2824] [Q&A]
PRETTY_NAME="Ubuntu 22.04.1 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.1 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy

qdrant 1.6.1
cargo +1.72.1 build --release --target=x86_64-unknown-linux-musl

= note: some `extern` functions couldn't be found; some native libraries may need to be installed or have their path specified
  = note: use the `-l` flag to specify native libraries to link
  = note: use the `cargo:rustc-link-lib` directive to specify the native libraries to link with Cargo (see https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorustc-link-libkindname)

error: could not compile `qdrant` (bin "qdrant") due to previous error

**Comment by timvisee** [comment]
Do you see any other error messages during compilation?

Also, what host architecture are you using. Are you on a x86_64 machine?

We currently don't support musl builds, because some of our dependencies are incompatible with this. Please see: https://github.com/qdrant/qdrant/issues/2656
